app/controllers/groups_controller.rb

```diff
@@ -4,8 +4,8 @@
 class GroupsController < Groups::ApplicationController # rubocop:disable Metrics/ClassLength
   layout :resolve_layout
   before_action :group, only: %i[edit show destroy update transfer]
-  before_action :context_crumbs, except: %i[index new create show]
   before_action :authorized_namespaces, except: %i[index show destroy]
+  before_action :context_crumbs, except: %i[index new create]
```

Comment:
**nitpick (blocking):** This before_action is not required anymore with your refactor

Reply:
The ```before_action``` was removed in both ```groups``` and ```projects``` controllers in 3ddabf0bdb2ffa66ba1cda3f0446ce4e21177ce8